pagopa/io-app
# chore: [IOPAE-1206] Replace `TextInput` with `SearchInput` in the services section

## Short description
This PR replaces `TextInput` with `SearchInput` in search screen and adds the pressable `SearchInput` in services home screen.

<details open><summary>Details</summary>

<p>

| SearchInput |
| - | 
| <video src="https://github.com/pagopa/io-app/assets/29163287/80e97455-0227-4322-ba06-f1471fc4f6ab" width="300"/> |
</p>
</details> 

## List of changes proposed in this pull request
- bump `@pagopa/io-app-design-system` to `1.38.0`
- added pressable `SearchInput` in services home screen
- replaced `TextInput` with `SearchInput` in search screen
- updated locales

## How to test
Using `io-dev-api-server`, navigate to the services tab. Check that the search works as expected.
